### DISCUSSION TITLE
Sensor not seeing traffic, even though bond0 is

### DISCUSSION THREAD

**Discussion by rosswakelin** [Security-Onion-Solutions/securityonion#6647] [Unsupported Versions] [answered]
We have installed the latest version from the ISO, onto the hardware we want to use for IDS sensors.  We have SPAN'ned traffic to that port, and the switch is showing a continuous 100-odd mb/s going to the port.  Running ifconfig on the sensor shows lots of received traffic.  Running tcpdump against bond0 as an interface shows lots of traffic.
The Grafana dashboard for the sensor shows zero traffic being received against the Monitor interface - but it does show traffic against the management port, so the manager/sensor connection is going ok.  All the Monitor interface stats are 0, including packets, drops etc.
CPU usage is about 2%, heaps of disk, 0% swap used, its almost as if the SO software can't read from the bond0, or even stats for the bond0.
The log count by node show a grand total of 9 events for the sensor, and they are all Suricata complaining about not enough (i.e. any) traffic seen on the monitor interface.
Anyone with any ideas?

**Comment by dougburks** [comment, marked as answer]
How many NICs do you have?

What kind of NICs are they?

Have you tried different NICs?

What version of Security Onion are you using?

Have you tried a fresh installation of the latest version?

**Reply by rosswakelin** [reply]
Hi there.
Well after somewhere between 24 and 48 hours, all the data started appearing in the Grafana screens, and a whole lot of Suricata alerts appeared in the Dashboard.  During that period, nothing had been done to the IDS box, no reboots, no updates, nothing. 
FYI, we are using a Supermicro 1U appliance for the IDS sensor, with Xeon processor, 16Gb memory, 500Gb SSD.  It has six 1Gb copper ports (one used for management) and two 10GB capable SFP+ ports.  We have added all 7 "non-management" ports to the bond0, and have one of the 1Gb ports connected to a switch span port for our testing.  The bond0 is now showing about 200Mb/s average, with no packet loss and about 12% CPU use.  We will load it up with more span ports over the next couple of days and see how it copes. 
In the meantime, I will close this thread.  Thanks for responding.
![Capture](https://user-images.githubusercontent.com/22168385/147151305-7ed5b474-b80e-4d6f-ba35-bf033745c682.PNG)
